style.css

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
I recommend not leaving empty spaces at the top of the CSS documents, as it may be outside the bounds of the styling help from linters.